pdf.js

```diff
@@ -495,6 +518,7 @@ PDFViewer.prototype = {
       return;
     if (this.loadCallback_)
       this.loadCallback_(this.loadState_ == LoadState.SUCCESS);
+    window.dispatchEvent(new Event('pdf-loaded'))
```

Comment:
Sounds good but the event and `loadCallback_` are doing the same job and we already use the latter, so it seems redundant to have both. The `page-title-updated` event is just used to set the `loadCallback_`.